### DISCUSSION TITLE
MyFbFramework 1.2.9

### DISCUSSION THREAD

**Discussion by XusinboyBekchanov** [XusinboyBekchanov/MyFbFramework#7] [General]
Version 1.2.9 (July 14, 2021)
- Fixed: Show windows on DPI sets
- Fixed: CheckBox problem in GroupBox
- Fixed: Default check property of CheckBox

<hr /><em>This discussion was created from the release <a href='https://github.com/XusinboyBekchanov/MyFbFramework/releases/tag/'>MyFbFramework 1.2.9</a>.</em>

**Comment by demosthenesk** [comment]
Is there in MyFbFramework a MDI Form which may contains other non-mdi forms?

**Reply by XusinboyBekchanov** [reply]
MDI form has not yet been implemented.

**Reply by XusinboyBekchanov** [reply]
Fixed: MDIForm and MDIChild form styles in Windows:
https://github.com/XusinboyBekchanov/MyFbFramework/commit/47af975d021cb7e2883f1b8aad5491b07ae644da

**Reply by XusinboyBekchanov** [reply]
Added: Initial changes for MDIForm and MDIChild in Linux:
https://github.com/XusinboyBekchanov/MyFbFramework/commit/a7bf5808d89d4478b5c28f7640018f03970c9d5b

**Reply by AntonioFS-1962** [reply]
Greetings from Sabadell-Barcelona-Spain.

Hello again.

I attach url of pdf with the steps followed according to instructions in GitHub. Also at the end I check that GTK3 is installed on my computer.

Thank you very much.
Antonio F.S.

https://drive.google.com/file/d/1hsm3ag56Rg8XkprPxEUdor7iEZ_2tWs2/view?usp=sharing

**Reply by XusinboyBekchanov** [reply]
I answered here:
https://github.com/XusinboyBekchanov/VisualFBEditor/discussions/83#discussioncomment-1385922

**Comment by AntonioFS-1962** [comment]


**Reply by XusinboyBekchanov** [reply]
I answered here:
https://github.com/XusinboyBekchanov/VisualFBEditor/discussions/83#discussioncomment-1385370
